TREP-LABS/realdrip-backend
# Send mail to verify email address after signup

#### What does this PR do?
- Adds an email service that can be used to send mails
- Use the email service to send mail to the user for email-address verification after signup

#### How can it be tested locally?
- Setup the app(some new environment variables are required, check `.env.examle`)
- Try signing up with a valid email address 
- After successful signup, you should receive an email notification to confirm your email address
- NB: The functionality of confirming the email address is not yet working, so if you click the button in the email to verify your email, don't be 😞 

#### Snapshot  
- Confirmation mail  
![Capture2](https://user-images.githubusercontent.com/26048536/65599281-6d657e80-df95-11e9-88f8-4639af8637ba.PNG)
